_pytest/test_server.py

```diff
@@ -5,35 +5,67 @@
 from __future__ import absolute_import
 
 import os
+import signal
 import tempfile
+import requests
 
 import pytest
 import time
+from multiprocessing import Semaphore
 
 import utilities
 from rasa_nlu.config import RasaNLUConfig
 import json
 import io
 
 from utilities import ResponseTest
-from rasa_nlu.server import create_app
+from rasa_nlu.server import RasaNLU
 
 
 @pytest.fixture(scope="module")
-def app(tmpdir_factory):
+def stub(tmpdir_factory):
```

Comment:
They are not really the same fixtures: they use different configs, one trains new models, and makes use of a component builder. But I agree there is some common code there.